packages/commonwealth/server/migrations/20240104232957-reaction-community-semantics.js

```diff
@@ -0,0 +1,116 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      await queryInterface.renameColumn('Reactions', 'chain', 'community_id', {
+        transaction,
+      });
+      await queryInterface.sequelize.query(
+        `
+        ALTER INDEX "OffchainReactions_pkey" RENAME TO "Reactions_pkey";
```

Comment:
Does capitalization matter here vs below?

Reply:
Logically no but it seems that the general pattern is to capitalize primary/foreign keys and unique indexes. That said it doesn't seem to be the same everywhere. Might be a good time to establish some kind of standard but shouldn't be a blocker for this.

I pushed a change to capitalize the foreign keys for now but left the indexes the same.